alembic/versions/e9d53a640d5d_add_start_end_datetimes_notifications.py

```diff
@@ -0,0 +1,32 @@
+"""add start end datetimes for notifications
+
+Revision ID: e9d53a640d5d
+Revises:
+Create Date: 2020-12-11 16:06:21.960486
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "e9d53a640d5d"
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column(
+        "notifications",
+        sa.Column("start_date", sa.DateTime(timezone=True), nullable=True),
+    )
+    op.add_column(
+        "notifications",
+        sa.Column("end_date", sa.DateTime(timezone=True), nullable=True),
+    )
```

Comment:
What is the expected migration path for a fresh database here? If I am correct, this implementation requires one to run the previous migration script first and then run alembic stuff afterwards. I do think we need to get a base migration first that mirrors the current state of the database and then start adding migrations to that